kyverno-integration/modules/audit/kyverno-pss-policies-audit.yaml

```diff
@@ -0,0 +1,33 @@
+kind: Application
+apiVersion: argoproj.io/v1alpha1
+metadata:
+  name: kyverno-pss-policies-audit
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/kyverno/kyverno
+    targetRevision: HEAD
```

Comment:
Can we pin it to a version? HEAD isn't an issue if we are syncing manifests from local storage, but in this case we are not. I'd like there to be a bit of assurance that this will keep working even after some potential breaking changes are introduced.

Reply:
Yeah I had it version pinned but didn’t see what the guidance on maintaining these was so went with the “current” approach

Reply:
Good point. We really need to come up with a guideline.

Reply:
hmm i may have to leave it at HEAD or switch to installing kyverno differently. for some reason the repo just specifies v0.0.0 in the Chart file

Reply:
https://github.com/kyverno/kyverno/tree/kyverno-chart-3.2.7/charts
Looks like 3.2.7 is the latest? https://artifacthub.io/packages/helm/kyverno/kyverno

Reply:
Let me see why it was bombing out locally when I was testing